docs/guides/agent/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Agent
+sidebar_position: 20
+---
+
+The zrok agent centralizes management of your public and private zrok shares and private frontends for accessing [private shares](/concepts/sharing-private.mdx). It provides a web-based console interface and changes how the `zrok share` and `zrok access` commands behave.
+
+## Tutorial
+
+Run the agent in the foreground.
+
+```bash
+zrok agent
+```
+
+In another terminal, open the console.
+
+```bash
+zrok agent console
+```
+
+You should see the agent UI in your default web browser.
+
+Start sharing a public share with the agent.
+
+```bash
+zrok share public 8080
+```
+
+```buttonless title="Output"
+token:"zje5x8p0k9pi" frontendEndpoints:"https://zje5x8p0k9pi.share.zrok.io"
+```
+
+You will see the new public share in the agent UI and you can access it at the public share URL.
+
+Reserve a private share for the agent to share.
+
+```bash
+zrok reserve private 8080 --closed --unique-name "myshare"
+```
+```buttonless title="Output"
+[   1.883]    INFO main.(*reserveCommand).run: your reserved share token is 'myshare'
+```
+
+Start sharing the reserved share with the agent.
+
+```bash
+zrok share reserved "myshare"
+```
+```buttonless title="Output"
+[   0.001]    INFO main.(*shareReservedCommand).shareAgent: starting
+token:"myshare" backendMode:"proxy" shareMode:"private" target:"http://127.0.0.1:8080"
+```
+
+You will see the new reserved share in the agent UI and you can access it by running `zrok access "myshare"` on another device where you have enabled the same zrok account, since the share was reserved with closed permission mode.
+
+### Running the Agent in the background
```

Comment:
There should also be a callout here for installing the Agent as a Windows service.